awesome_cart/awesome_cart/doctype/awc_item/awc_item.py

```diff
@@ -11,3 +11,31 @@ class AWCItem(Document):
 
 	def on_update(self):
 		clear_cache()
+
+
+@frappe.whitelist()
+def query_items_with_awc_items(doctype, txt, searchfield, start, page_len, filters):
+	return frappe.db.sql("""
+		select a.product_name as name, i.item_name as description from `tabAWC Item` a, `tabItem` i
+			where 
+				a.product_name = i.name
+				and a.catalog_visible=1
+				and (
+					a.product_name like %(txt)s
+					or i.name like %(txt)s
+					or i.item_code like %(txt)s
+					or i.item_name like %(txt)s
+				)
+				and (i.variant_of is null 
+				or i.variant_of = '')
+			order by
+				product_name
+			limit %(start)s, %(page_len)s
+		""".format(**{
+			'key': searchfield
```

Comment:
What does this replace exactly?

Reply:
This is used on the related products filter. So that it only lists items which we have a matching AWC Item for.

Reply:
Exhibit A: